core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -479,4 +481,16 @@ public FeaturesConfig setForceSpillingJoin(boolean forceSpillingJoin)
         this.forceSpillingJoin = forceSpillingJoin;
         return this;
     }
+
+    public boolean isFteExchangeEncryptionEnabled()
+    {
+        return fteExchangeEncryptionEnabled;
+    }
+
+    @Config("fte-exchange-encryption-enabled")
```

Comment:
I like the shorter version of property but it is kinda first time we would use acronyms in config.
On the other hand, the `fault-tolerant-execution-` prefix is super annoying.

@martint do you think it is ok to use `fte-` here (and rename other properties this way?). I thought about alternative but naming is hard.

Reply:
I'd prefer not to use acronyms. It's not a big deal if the name is long (alternatively, we could try to come up with a shorter, more descriptive name). However, it shouldn't be a prefix such that properties are all named `fault-tolerant-execution-xxxx`. Instead, we should turn it into a hierarchical property: `fault-tolerant-execution.xxx` (note the `.`)